pkg/term/proxy.go

```diff
@@ -31,48 +32,54 @@ func NewEscapeProxy(r io.Reader, escapeKeys []byte) io.Reader {
 	}
 }
 
-func (r *escapeProxy) Read(buf []byte) (int, error) {
-	nr, err := r.r.Read(buf)
+func (r *escapeProxy) Read(buf []byte) (n int, err error) {
```

Comment:
Not a huge fan of named output variables; it's easy to inadvertently mask them and introduce bugs; perhaps we can slightly refactor the code to not require them

Reply:
I'd have to make variable declarations at the beginning of the function, this wouldn't look better IMO, I don't see an alternative with the current implementation.

Reply:
Hmm, 🤞 that we won't have to touch this again! so ok for named output variable here